superset/db_engine_specs/presto.py

```diff
@@ -29,13 +29,16 @@
 from sqlalchemy.engine.result import RowProxy
 from sqlalchemy.sql.expression import ColumnClause
 
+from superset import app
 from superset.db_engine_specs.base import BaseEngineSpec
 from superset.exceptions import SupersetTemplateException
 from superset.models.sql_types.presto_sql_types import type_map as presto_type_map
 from superset.utils import core as utils
 
 QueryStatus = utils.QueryStatus
 
+PRESTO_VERSION = app.config.get("PRESTO_VERSION")
```

Comment:
@etr2460 I don't think this is correct as there may be multiple Presto databases. Either this should be defined in the Database CRUD model (preferred) or this variable needs to be a dictionary keyed by database ID.

Reply:
Eek, that's true... This should be added to the database model

Reply:
There may be a way of determining this from the connection, though regardless it probably should be stored either in the model or cached.

Reply:
yeah, i tried to figure out a way to from the connection, but couldn't find one. I've added it to the extra params in the database model now, as i didn't think it should use a new column (since we're only using it with presto dbs right now)